fakenect/record.c

```diff
@@ -201,6 +203,70 @@ void print_mode(const char *name, freenect_frame_mode mode) {
 	   mode.framerate, mode.is_valid);
 }
 
+static void write_device_info(freenect_device *dev)
+{
+	JSON_Value *js = json_value_init_object();
+	JSON_Object *dev_js = json_object(js);
+
+	JSON_Value *reg_info_val = json_value_init_object();
+	JSON_Object *reg_info = json_object(reg_info_val);
```

Comment:
Does this one need to be freed as well?

Reply:
I'm fairly sure parson will recursively free the objects given the top-level one to free at the end, but can double check that

Reply:
ok, just to confirm; `json_value_free()` is recursive, so since that object got linked into the hierarchy of the top-level `js` value then it will also be freed.